CHANGES.md

```diff
@@ -30,6 +30,8 @@
 
   + Fix placement of comments on constants (#1383) (Guillaume Petiot)
 
+  + Remove trailing space after expression when followed by an attribute and make attributes more obvious at the beginning of a line (#1382) (Guillaume Petiot)
```

Comment:
Would it be accurate to write something like:
```suggestion
  + Remove trailing space after expression when followed by an attribute and break before attributes attached to multi-line phrases (#1382) (Guillaume Petiot)
```
and leave the more obvious part to the interpretation of readers?